charts/k8s-monitoring/templates/alloy_config/_processors.alloy.txt

```diff
@@ -212,6 +212,40 @@ otelcol.processor.transform "default" {
   }
 }
 
+
+{{- if and .Values.traces.enabled .Values.traces.receiver.tailsampling.policies }}
+otelcol.processor.tail_sampling "default" {
+  {{- range $policy := .Values.traces.receiver.tailsampling.policies }}
+  policy {
+    name    = {{ $policy.name | quote }}
+    type    = {{ $policy.type | quote }}
+  {{- if and (eq $policy.type "latency") (hasKey $policy "latency") }}
+    latency = {
+      threshold_ms = {{ $policy.latency.thresholdMs | int }},
+    }
+  {{- end }}
+
+  {{- if and (eq $policy.type "string_attribute") (hasKey $policy "stringAttribute") }}
+    stringAttribute = {
+      key = {{ $policy.stringAttribute.key | quote }}
+      values = [
+      {{- range $value := $policy.stringAttribute.values }}
+        {{ $value }},
+      {{- end }}
+      ]
+      enabled_regex_matching = {{ $policy.stringAttribute.enabledRegexMatching | default true }}
```

Comment:
I think both `enabled_regex_matching` and `invert_match` can be wrapped in a {{- if }} {{- end }} block, since they're not required in the `string_attribute` block spec.

Also, `enabled_regex_matching` should default to `false`.